### ISSUE TITLE
Error using forms validation on Base master: Uncaught TypeError: $(...).validate is not a function

### ISSUE THREAD

**Issue by teknologist** [themeteorchef/base#178]
Hi,

Just using latest base master.
Did a meteor update and updated npm packages to their latest versions.
Then did a meteor npm install to fetch all node_modules.

When I try to login or signup:

```
signup.js:35 
Uncaught TypeError: $(...).validate is not a function 
validate @ signup.js:35
handleSignup@ signup.js:74
componentDidMount@ signup.js:8
invokeComponentDidMountWithTimer @ ReactCompositeComponent.js:60
_assign.notifyAll @ CallbackQueue.js:67
...
```

Seems the jquery-validation import is not kicking in..

Any idea ?

My devDeps:

```
"devDependencies": {
    "chimp": "^0.40.4",
    "eslint": "^3.4.0",
    "eslint-config-airbnb": "^10.0.1",
    "eslint-plugin-import": "^1.14.0",
    "eslint-plugin-jsx-a11y": "^2.2.0",
    "eslint-plugin-meteor": "^4.0.0",
    "eslint-plugin-react": "^6.2.0",
    "faker": "^3.1.0"
  }
```

and my prdouction deps:

```
"dependencies": {
    "bcrypt": "^0.8.7",
    "bootstrap": "^3.3.7",
    "jquery": "^3.1.0",
    "jquery-validation": "^1.15.1",
    "react": "^15.3.1",
    "react-addons-pure-render-mixin": "^15.3.1",
    "react-bootstrap": "^0.30.3",
    "react-dom": "^15.3.1",
    "react-komposer": "^1.13.1",
    "react-router": "^2.7.0",
    "react-router-bootstrap": "^0.23.1"
  }
```

**Comment by teknologist** [comment]
I tried downgrading jquery to 2.2.4 just in case...but it doesn't change anything. Issue is still there...

**Comment by themeteorchef** [comment]
This should be patched in the 4.8.0 release!
